packages/css/src/components/skip-link/README.md

```diff
@@ -13,9 +13,9 @@ After appearing, it pushes the rest of the page down.
 ## Guidelines
 
 - Place the Skip Link as the first element in the [Page](/docs/components-containers-page--docs) container.
-- Target the Skip Link to the main content.
-  On an article page, for example, it could be the title of the article.
-  On a search page, it could be the search field.
-- Set `id="example-id"` on the container of that element and then use `href="#example-id"` on the Skip Link.
+- Direct the Skip Link to the main content area on the Page.
+  Typically, this is the `main` HTML element.
+  Repeated navigation blocks such as Breadcrumbs and back Links do not belong to the main content area.
+- Set `id="main"` on the `main` HTML element and then use `href="#main"` on the Skip Link.
```

Comment:
Our websites are typically in Dutch. Urls that use slugs of their page title will then also be in Dutch. Let’s advise to use Dutch as well for these fragments. [Urls should be hackable](https://www.nngroup.com/articles/url-as-ui/).

How about `#inhoud`, even though we then pick the other word of ‘main content’? `#hoofd` is weird.